packages/paste-theme/__tests__/__snapshots__/generateThemeFromTokens.spec.ts.snap

```diff
@@ -80,6 +80,7 @@ Object {
     "sizeSquare90": "1px",
   },
   "iconSizes": Object {
+    "sizeIcon05": undefined,
```

Comment:
question: I know this is a generated snapshot but is this `undefined` what's supposed to be here? it shouldn't be 1px like the rest?

Reply:
I wonder if this is indicative of a bug? I feel like it should be something.